automation/taskcluster/decision_task_release.py

```diff
@@ -50,8 +50,6 @@ def generate_build_task(version):
     checkout = ("git fetch origin --tags && "
                 "git config advice.detachedHead false && "
                 "git checkout {}".format(version))
-    bintray_publishing = (" && python automation/taskcluster/release/fetch-bintray-api-key.py"
-                          " && ./gradlew bintrayUpload --debug")
 
     assemble_task = 'assembleRelease'
     scopes = [
```

Comment:
I think we can trash this scope as well from:

1. here 
2. from the [TC client.](https://tools.taskcluster.net/auth/roles/repo%3Agithub.com%2Fmozilla-mobile%2Fandroid-components%3Arelease)
3. from the main `.taskcluster.yml` file [here](https://github.com/mozilla-mobile/android-components/blob/master/.taskcluster.yml#L90)

Reply:
Great catch! I just removed it from 1 and 3. I'll remove from the TC client after we shipped the first release without bintray.

Reply:
Makes perfect sense, ideal to have a fallback just in case things go wrong! 👍